### DISCUSSION TITLE
is there a version that will work with React Three Fiber?

### DISCUSSION THREAD

**Discussion by carmandale** [srcnalt/3d-profile-avatars#3] [Q&A] [answered]
I can't seem to make it work.

**Comment by srcnalt** [comment, marked as answer]
Hi, this package is based on React Three Fiber. The dependencies are as follows:

```json
{
  "@react-three/drei": "^7.2.2",
  "@react-three/fiber": "7.0.5",
  "@types/three": "^0.130.1",
  "three": "^0.130.1"
}
```

If you mean [`react-three-fiber`](https://www.npmjs.com/package/react-three-fiber) (which is now obsolete) instead of `@react-three/fiber` I do not have plans to convert it to older versions.

Otherwise could you provide more detail on the problem you are having?
